tests/Reader/XLSX/ReaderTest.php

```diff
@@ -562,12 +562,13 @@ public function testReadShouldBeAbleToProcessEmptySheets(): void
         self::assertSame([], $allRows, 'Sheet with no cells should be correctly processed.');
     }
 
-    public function testReadShouldSkipFormulas(): void
+    public function testReadShouldReadFormulas(): void
     {
         $resourcePath = TestUsingResource::getResourcePath('sheet_with_formulas.xlsx');
 
         $allRows = [];
         $options = new Options();
+        $options->SHOULD_READ_FORMULAS = true;
```

Comment:
The primary goal of this library is to be idempotent: reading and re-writing a file should yield the same result.

The default value here should be false, hence this diff must be reverted.

Reply:
Thanks for looking into this. I agree with you and no longer think this extra option need to be added at all. I'll close the PR.